cogs/tickets.py

```diff
@@ -156,21 +164,61 @@ def topic_check(m):
             icon_url=av,
             text="IsThicc Tickets"
         )
-        await ticket.send(content="<@!796953153010532362>", embed=opened)
+        await ticket.send(content="<@&796953153010532362>", embed=opened)
 
         await self.db.execute('INSERT INTO tickets VALUES(' + str(ticket.id) + ', ' + str(reaction_user.id) + ', true)')
-        # parse as str in query, but the database will parse it as whatever you chose it to be in phpmyadmin.
 
-        #   # Channel id(ticket.id):                 ticket_id
-        #   # Ticket opener id(reaction_user.id):    user_id
-        #   # Open or closed(bool: true):            open_close
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    @commands.command()
+    async def add(self, ctx, member: discord.Member):
+        av = self.bot.user.avatar_url
+
+        if not ctx.guild.get_role(796953153010532362) in ctx.author.roles:
+            await ctx.message.delete()
+            return await ctx.channel.send(embed=discord.Embed(
+                title="You dont have permission to add a user to this ticket!",
+                colour=discord.Colour.red()
+            ), delete_after=10)
+
+        try:
+            r = await self.db.execute('SELECT * FROM tickets WHERE channel_id = ' + str(ctx.channel.id))
+        except Exception:
+            not_ticket = em(
+                title="Sorry!",
+                description="Sorry! This channel isn't a ticket. Please use this command again in a ticket.",
+                colour=discord.Colour.red(),
+                timestamp=datetime.utcnow()
+            )
+            not_ticket.set_footer(
+                icon_url=av,
+                text="IsThicc Tickets"
+            )
+            return await ctx.send(embed=not_ticket)
+
+        await ctx.channel.set_permissions(member, send_messages=True)
+        await ctx.send(embed=discord.Embed(
+            title=f"Added {member.name} to the ticket",
+            colour=discord.Colour.green(),
+            timestamp=datetime.utcnow()
+        ).set_footer(
+            icon_url=av,
+            text="IsThicc Tickets"
+        ))
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     @commands.command()
     async def close(self, ctx):
-
         av = self.bot.user.avatar_url
+        user = await self.db.execute("SELECT user_id FROM tickets WHERE channel_id = " + str(ctx.channel.id))
```

Comment:
This is not caught if it errors, unless it is in the main error handler.

Reply:
it checks for user id, the only other way the query could fail is if the its not connected to the database

Reply:
I'm just dumb. I was thinking of something else.